WebRoot/templates/prefs/Pages.template

```diff
@@ -674,23 +674,28 @@ and the container element is replaced with that control.
 <template id='prefs.Pages#TrustedList'>
 	<table role="presentation" class='ZPropertySheet' cellspacing='6'>
 		<tr>
-			<td id='${id}_EMAIL_ADDRESS' tabindex=0></div></td>
-			<td id='${id}_ADD_BUTTON' valign=top tabindex=0></td>
+			<td>
+				<table>
```

Comment:
As discussed, we need a wrapper for both email input container and add icon. So we have a td and inside that the new table.
--